src/components/themed/ThemedTicker.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import * as React from 'react'
+import { Platform } from 'react-native'
+import Ticker from 'react-native-ticker'
+
+import { useTheme } from '../services/ThemeContext.js'
+
+type Props = {
+  children: React.Node,
+  style: any,
+  numberOfLines?: number,
+  ellipsizeMode?: string,
+  durations?: number,
+  [x: string]: any
+}
+
+export const ThemedTicker = ({ children, style, duration, ...props }: Props) => {
+  const theme = useTheme()
+
+  // TODO: The nature of the Ticker compoenent does not apply textProps to the string as a whole so these propds dont work.
+  // May not be able to have the text automatically adjust to fit with this ticker. If this is needed, may need more creative solution.
```

Comment:
Edgetext autosizes to fit its container width and will honor the number of lines prop. 
ThemedTicker does not offer this out of the box. 

Determine if this will end up being an issue or not.

Reply:
should be ok

Reply:
You sure we wont have any people on the wallet app the are investing 23423857349573094875390875340985? :p 

Just kidding, that makes sense! Appreciate your quick feedback! will move forward with this information